test/Sema/availability_literals_inferred.swift

```diff
@@ -0,0 +1,23 @@
+// RUN: %target-typecheck-verify-swift -swift-version 5
+
+// REQUIRES: OS=macosx
```

Comment:
At first glance it doesn't seem like this test should require `macosx` since the availability annotations specify universal unavailability.